de/de.py

```diff
@@ -0,0 +1,79 @@
+"""module changes the language of the table names in CSV Files and transfers
```

Comment:
no reading of files from s3, nor writing to the database, which was one of the main requirements of this task